src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -295,7 +298,10 @@ describe('processBulkUpload', () => {
 			getMockJobHelpers(),
 		);
 		const updatedBulkUpload = await loadBulkUpload(bulkUpload.id);
-		expect(updatedBulkUpload.status).toEqual(BulkUploadStatus.FAILED);
+		expect(updatedBulkUpload).toMatchObject({
+			status: BulkUploadStatus.FAILED,
+			fileSize: 93,
```

Comment:
I'll just leave one comment, but it affects multiple tests: `93` isn't the correct value for all the test CSVs. Need to update them to match the correct expected value. (I assume this is 🍝copypasta🍝 from the `validCsvTemplate.csv` test.)